accumulator/hasher.go

```diff
@@ -15,12 +11,11 @@ type hashableNode struct {
 // so the data is "there" in a static structure so I can use pointers to it
 // and it won't move around.  hopefully.
 
-func (n *hashableNode) run(wg *sync.WaitGroup) {
```

Comment:
since this is now a one-liner used in only one location, wouldn't it be better to just inline it? since there is no longer even a go routine, i think the name "run" doesn't add any value at all.

Reply:
Hmm yeah good call I thought maybe the compiler would inline it but it doesn't. Will change